pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 0.1.0+1
+version: 0.2.0+2
```

Comment:
Hmm, We don't have a release version for `0.1.0` yet, Not sure, Do we need to bump the version to `0.2.0` 🤔

Reply:
@Wadeewee I'd like to carry over the 0.1.0 features to 0.2.0 and release, which is okay, and we do that with client projects sometimes.

Reply:
Got it @AVI5HEK 🚀